src/plugins/plot/legend/PlotLegendItemCollapsed.vue

```diff
@@ -103,8 +105,18 @@ export default {
         }
     },
     mounted() {
+        eventHelpers.extend(this);
```

Comment:
IS there any value in making this a mixin that is used by *Collapsed and *Expanded?

Reply:
Yes and no. There is some overlap, but the initialize() method is slightly different. If this changes in the future I think there might be value.